docs/conf/portland/2018/sponsors/prospectus.rst

```diff
@@ -163,52 +174,41 @@ The **Keystone** package costs **$14,000**.
 Other Sponsorship Opportunities
 ===============================
 
-The following à la carte offerings are available either independently or
+The following a la carte offerings are available either independently or
 combined with one of the previous packages to increase visibility at the event.
 
-Financial Assistance
---------------------
+Job Fair
+--------
 
-Write the Docs is always trying to make our community more diverse and
-inclusive. Talk to us about sponsoring travel and/or registration expenses for
-an attendee who wouldn’t otherwise be able to attend.
+Our job fair is a wonderful place to connect with the talented Write the Docs attendees.
+We'll have over 400 people in attendance for our conference,
+and many of those people will come to our job fair to look at possible new directions in the industry.
 
-Lanyard
--------
+**$1,000** (a la carte price, also included or discounted with other sponsorship packages)
 
-Provide your branded lanyards for our name tags.
+- **Logistics**: The Job Fair will be Tuesday morning in Lola's Room. It will run from 9am-11:30pm. Setup at 8am.
+- **Layout**: Every participant will have a six-foot table and 2 chairs set reserved for them. We do not use booths during the Job Fair. You are welcome to have a standing banner or customized tablecloth. Giveaways/flyers are welcome.
```

Comment:
s/set// ?

Reply:
Is there any thing we need to tell them they *can't do*?

Reply:
- We do not use booths ever ;)
- I would limit the swag presence, so for example 1 roll-up banner and any swag they can fit on the table, but no inflatable arm-flailing tube men.
- The content services director asked me if it means RH should send a recruiter, and I recommended that if he wants to send one recruiter or manager, and have them tag-team with a tech staff member that would be better than just sending a bunch of recruiters.
- No table/booth babes
- Maybe explicitely state that this is the only place where people can engage in active recruiting, and the CoC will not tolerate if they let loose during the rest of the conf